packages/nimble-components/src/button/tests/button.mdx

```diff
@@ -22,16 +22,93 @@ import * as buttonStories from './button.stories';
 
 <br />
 
+## Styling
+
+<br />
+
+    ### Appearances
```

Comment:
Could you make the indentation in your changes more consistent? I think we don't usually bother to indent various levels of headers and their content. It can all just be left aligned.